src/main/java/data/SheetsConverter.java

```diff
@@ -17,9 +17,7 @@
 import com.google.api.services.sheets.v4.model.*;
 import java.io.IOException;
 import java.security.GeneralSecurityException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
```

Comment:
As a general practice, we avoid importing .*

I'd prefer to see each import explicitly